### PR TITLE
Liveness, readiness probes

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2178

### CODE REVIEW

File: cmd/chainbridge/main.go [hunk 1]
```diff
@@ -227,6 +227,12 @@ func run(ctx *cli.Context) error {
 		h := health.NewHealthServer(port, c.Registry, int(blockTimeout))
 
 		go func() {
+			http.HandleFunc("/liveness", func(w http.ResponseWriter, r *http.Request) {
```
Comment (AndreiNavoichyk) [inline cmd/chainbridge/main.go]: Update endpoints according to the tech design

File: cmd/chainbridge/main.go [hunk 2]
```diff
@@ -227,6 +227,12 @@ func run(ctx *cli.Context) error {
...
+				fmt.Fprintf(w, "Liveness probe is ok")
+			})
+			http.HandleFunc("/readiness", func(w http.ResponseWriter, r *http.Request) {
```
Comment (AndreiNavoichyk) [inline cmd/chainbridge/main.go]: Update endpoints according to the tech design

File: cmd/chainbridge/main.go [hunk 3]
```diff
@@ -227,6 +227,12 @@ func run(ctx *cli.Context) error {
...
+				fmt.Fprintf(w, "Rediness probe is ok")
```
Comment (AndreiNavoichyk) [inline cmd/chainbridge/main.go]: Fix typo Re**a**diness

File: cmd/chainbridge/main.go (hunk 2 above)
Comment (DmitriyMolch) [reply cmd/chainbridge/main.go]: fixed

File: cmd/chainbridge/main.go (hunk 3 above)
Comment (DmitriyMolch) [reply cmd/chainbridge/main.go]: fixed

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**